### PR TITLE
[ADD] project_category: module to categorize projects

### PR DESCRIPTION
**Pull request by cubells**
cc @Tecnativa

### CODE REVIEW

File: project_category/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2015 ADHOC SA (http://www.adhoc.com.ar)
+# Copyright 2017 Tecnativa - Pedro M. Baeza
+
+{
+    'name': 'Project Category',
+    'version': '11.0.1.0.0',
+    'category': 'Project',
+    'author': 'ADHOC SA,'
```
Comment (pedrobaeza) [inline project_category/__manifest__.py:8]: OCA missing

File: project_category/i18n/es.po [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+#. module: project_category
+#: model:ir.ui.view,arch_db:project_category.project_type_form
+msgid "Add a description..."
+msgstr "Añadir una descripción ..."
+
+#. module: project_category
+#: model:ir.model.fields,field_description:project_category.field_project_type_create_uid
+msgid "Created by"
+msgstr "Created by"
```
Comment (pedrobaeza) [inline project_category/i18n/es.po]: Missing translations

File: project_category/readme/DESCRIPTION.rst [hunk 3]
```diff
@@ -0,0 +1 @@
+This module add the category type fields in project.
```
Comment (pedrobaeza) [inline project_category/readme/DESCRIPTION.rst]: s/add/adds

File: project_category/readme/USAGE.rst [hunk 4]
```diff
@@ -0,0 +1,4 @@
+* Go to **Project > Configuration > Project Type** and add some items in order to
```
Comment (pedrobaeza) [inline project_category/readme/USAGE.rst:1]: Better to be `Project Types` or `Project Categories` as the module name.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: project_category/models/project_type.py [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+class ProjectType(models.Model):
+    _name = 'project.type'
+    _description = 'Project Type'
+
+    name = fields.Char(
+        string='Name',
+        required=True
+    )
+    description = fields.Text('Description')
```
Comment (oihane) [inline project_category/models/project_type.py]: Shouldn't they be translatable?

Comment (pedrobaeza) [reply project_category/models/project_type.py]: Yes, good idea, but both.

**Comment by cubells** [comment]
@oihane done!

**Review by oihane — APPROVED** [review]
Looks great

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**